src/Modules/Events/ResourceListComponent.js

```diff
@@ -43,7 +46,7 @@ const ResourceListComponent = ({ dataToChild }) => {
             display_url,
             logo,
             id,
-            foodbank_texts
+            foodbank_texts,
```

Comment:
Is that trailing comma an issue?

Reply:
Ha no. I have an auto format on save that is tailored to my company's linting rules. 

Some people like a comma on the last line in Javascript so it's easy to add another. It really doesn't mean anything like it does in JSON or ruby.

Reply:
Thanks Sean. I've lived a lot of SQL, so I'm highly trained to look for trailing commas. Thanks for the explanation.